etna/transforms/timestamp/holiday.py

```diff
@@ -70,3 +72,12 @@ def transform(self, df: pd.DataFrame) -> pd.DataFrame:
         df = df.join(encoded_df)
         df = df.sort_index(axis=1)
         return df
+
+    def get_regressors_info(self) -> List[str]:
+        """Return the list with regressors created by the transform.
+        Returns
+        -------
+        :
+            List with regressors created by the transform.
+        """
+        return [self.out_column]  # type: ignore
```

Comment:
To fix the problem with typing you should remove this line `self.out_column = out_column` from the constructor and fix the definition on the next line. This happens because mypy think that out_column might be None, but in fact it is not.